docs/install/install-kn.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Installing the `kn` Client"
+linkTitle: "Installing the `kn` Client"
+weight: 10
+type: "docs"
+---
+
+## Install `kn` using the nightly executable binary
+To install the `kn` Client, you must download the executable binary for your system. Links to the latest nightly executable binary are available here:
+  - [macOS](https://storage.googleapis.com/knative-nightly/client/latest/kn-darwin-amd64)
+  - [Linux](https://storage.googleapis.com/knative-nightly/client/latest/kn-linux-amd64)
+  - [Windows](https://storage.googleapis.com/knative-nightly/client/latest/kn-windows-amd64.exe)
+
+You must place the executable binary in your system path, and make sure that it is executable.
+
+## Install `kn` using Go
```

Comment:
I don't think we should assume that folks have Go installed. Or, if we are going to assume that, we should mention it in a "Before you begin" section.